src/mpriscontroller.cpp

```diff
@@ -979,16 +979,18 @@ void MprisControllerPrivate::onServiceVanished(const QString &service)
     }
 
     m_availableClients.removeOne(client);
-    m_otherPlayingClients.removeOne(client);
     Q_EMIT parent()->availableServicesChanged();
```

Comment:
Should this signal be emitted after updating the current client?

Reply:
Shouldn't matter, but I guess it could be better to start emitting only after everything has its final value already.